src/app/globals.css

```
@@ -55,6 +55,22 @@
   }
 }
 
+@font-face {
+  font-family: 'NeoDunggeunmo';
```

Stylelint (17.9.0)

[error] Expected no quotes around "NeoDunggeunmo" (font-family-name-quotes)

(font-family-name-quotes)